tests/test_core.py

```diff
@@ -312,3 +312,30 @@ def test_generate_source(self):
         rs = [ts for (ti,ts) in r] 
         for (is1, is2) in zip(rs0, rs):
             compare_mstates(self, is1, is2)
+
+class TestInsertElemInModelFlame(unittest.TestCase):
+    def setUp(self):
+        testfile = os.path.join(curdir, 'lattice/test_0.lat')
+        self.testfile = make_latfile(testfile)
+
+    def test_insert_in_modelflame(self):
+        latfile = self.testfile
+        fm = ModelFlame(latfile)
+        r0,s0=fm.run(to_element=6)
+
```

Comment:
To this line, what is the result of `fm.get_element(index=5)`?

Reply:
The result of `fm.get_element(index=5)` in that point is,
`{name:'LS1_CA01:BPM_D1129', 'type': 'bpm'}`.
So, I checked `index=6` element parameters after the insertion is the same as the above.
https://github.com/kryv/flame-utils/blob/5f2e31a9065813cee240c6d013b884680fbf8baf/tests/test_core.py#L334-L337

Reply:
I would suggest to save `fm.get_element(index=5)[0]` as a `econf_before_insertion`, then refactor line 336-337 by replacing hardcoded strings.